client/dashboard/domains/domain-connection-setup/components/dns-propagation-progress-bar-style.scss

```
@@ -0,0 +1,10 @@
+@import '@wordpress/base-styles/variables';
+
+.dns-propagation-progress-bar {
+	width: 100%;
+
+	/* Change the progress bar color to the theme color */
+	& > div {
+		background-color: var( --wp-admin-theme-color );
+	}
```

```suggestion
	--wp-components-color-foreground: var(--wp-admin-theme-color);
	}
```
I found this pattern in MSD, maybe we should use it for consistency?

---

Good call! I've updated the code